src/Style/Label.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+const Label = styled.text`
```

Comment:
Sorry I'm not familiar with styled.text. Which HTML tag does it correspond to? Did you want to use styled.label?

Reply:
Also, while using this component chrome dev tools throws a warning

Reply:
Yes, I meant to use label. Probably I was doing some test with Text and forgot to put Label back. Thanks, Victor!